.github/CODEOWNERS

```diff
@@ -41,6 +41,7 @@
 airflow-core/src/airflow/ui/public/i18n/locales/ar/ @shahar1 @hussein-awala # + @ahmadtfarhan
 airflow-core/src/airflow/ui/public/i18n/locales/de/ @jscheffl # + @TJaniF @m1racoli
 airflow-core/src/airflow/ui/public/i18n/locales/he/ @eladkal @shahar1 @romsharon98 # +@Dev-iL
+airflow-core/src/airflow/ui/public/i18n/locales/it/ @bbovenzi # + @aoelvp94
```

Comment:
I can't be the codeowner. I don't know Italian.

Reply:
Would be the same for me... just my daughter has Italian as third language in School :-D Does this account?

Do we have any native speaker as codeowner? Having Italian translation would be cool but we would need some maintainer...

Reply:
Yes, I think this is acceptable for you... but we would need somebody else to be able to review from content perspective if one other raises a PR.